code/Test_definitions/sim-swap-subscriptions.feature

```
@@ -306,39 +331,24 @@ Feature: CAMARA sim swap subscriptions  API, v0.1.1
 # Error Code 403
 ##################
 
-  @sim_swap_subscription_creation_60_phone_number_token_mismatch
+  @sim_swap_subscription_retrieval_61_phone_number_token_mismatch

```

@bigludo7 I checked with Pedro regarding the SUBSCRIPTION_MISMATCH error because I wasn't fully sure if were right about the meaning. And spoiler: we weren't.
So this error is designed to cover the scenario where an API Client requests the creation of an event subscription for an event type for which it does not have permission. The token is valid and the client can consume the endpoint, but only for the events that were requested in the authentication.

Maybe with an example for device status this is clearer. There we have a scope for roaming on and another one for roaming off. If the token was generated for roaming on and the client requests the creation of a subscription for roaming off, there we should return the SUBSCRIPTION_MISMATCH error.

And this is why it is only present in the POST for creation

---

@fernandopradocabrillo Whaou got it ! now I remind ! - Kudos to Pedro
So I remove the tests for GET & DELETE
I'm not sure we need a Test for POST as we have only one event type.